concent_api/core/tests/utils.py

```diff
@@ -204,6 +205,16 @@ def _test_database_objects(
             self.assertEqual(ReceiveStatus.objects.last().delivered,        receive_delivered_status)
             self.assertEqual(ReceiveStatus.objects.last().message.task_id,  task_id)
 
+    def _test_subtask_state(self, task_id, subtask_id, subtask_state, provider_key, requestor_key, next_deadline = None):
+        last_subtask = Subtask.objects.last()
+        self.assertEqual(last_subtask.task_id,              task_id)
+        self.assertEqual(last_subtask.subtask_id,           subtask_id)
+        self.assertEqual(last_subtask.state,                subtask_state)
+        self.assertEqual(last_subtask.provider.public_key,  provider_key)
+        self.assertEqual(last_subtask.requestor.public_key, requestor_key)
+        if next_deadline is not None:
+            self.assertEqual(last_subtask.next_deadline.timestamp(), next_deadline)
+
```

Comment:
How about other points from task description ? We also would like to check
* how many new StoredMessages were created since last step (similar what is currently done for MessageAuth in auth tests).
* how many new clients were created since last check (I think its better then checking total count of them).
* type, task_id and subtask_id of all newly created StoredMessages.